CHANGELOG.md

```diff
@@ -19,13 +20,43 @@
 
 All notable changes to this project will be documented in this file.
 
+## 1.1.0
+* ### Merged Pull Requests
+    * ...
+
+* ### Resolved Issues
+    * ...
+
+* ### Known Issues
+    * ...
+
 ## 1.0.0
 * ### Merged Pull Requests
-    *  ...
+    * Add support for strain calibration (#468 #469)
+    * Add support for DAQmx calibration info properties (#473)
+    * Add support for WaitForValidTimestamp to nidaqmx-python (#479)
+    * Fix raw-read Function Names Spacing Issue (#492)
+    * Update libtime (#504)
+    * Update AI Bridge Default Argument Values (#509)
+    * Improve error reporting in some cases (#550)
+    * Automate Driver Install Experience within nidaqmx Module (#512 #582)
+    * Rewrote examples (#566 #575 #573 #581 #583 #587 #570 #589 #592 #594)
```

Comment:
This list is not in "Title Case", so don't capitalize common nouns.
```suggestion
    * Fix raw-read function names spacing issue (#492)
    * Update libtime (#504)
    * Update AI bridge default argument values (#509)
    * Improve error reporting in some cases (#550)
    * Automate driver install Experience within nidaqmx module (#512 #582)
    * Rewrote examples (#566 #575 #573 #581 #583 #587 #570 #589 #592 #594)
```

Reply:
There are updated as per suggestions.